cirq-google/cirq_google/devices/google_noise_properties.py

```diff
@@ -66,14 +83,101 @@ class GoogleNoiseProperties(devices.SuperconductingQubitsNoiseProperties):
             for that gate. Defaults to no-op for all gates.
     """
 
-    fsim_errors: Dict[noise_utils.OpIdentifier, cirq.PhasedFSimGate] = field(default_factory=dict)
+    fsim_errors: Dict[noise_utils.OpIdentifier, cirq.PhasedFSimGate] = dataclasses.field(
+        default_factory=dict
+    )
 
     def __post_init__(self):
         super().__post_init__()
 
         # validate two qubit gate errors.
         self._validate_symmetric_errors('fsim_errors')
 
+    def override(
+        self,
+        *,
+        gate_times_ns: Union[None, float, Dict[Type['cirq.Gate'], float]] = None,
+        t1_ns: Union[None, float, Dict['cirq.Qid', float]] = None,
+        tphi_ns: Union[None, float, Dict['cirq.Qid', float]] = None,
+        readout_errors: Union[None, Sequence[float], Dict['cirq.Qid', Sequence[float]]] = None,
+        gate_pauli_errors: Union[
+            None,
+            float,
+            Dict[Union[Type['cirq.Gate'], noise_utils.OpIdentifier], float],
+        ] = None,
+        fsim_errors: Union[
+            None,
+            'cirq.PhasedFSimGate',
+            Dict[Union[Type['cirq.Gate'], noise_utils.OpIdentifier], 'cirq.PhasedFSimGate'],
+        ] = None,
+    ):
+        """Returns a copy of this object with the given fields overridden.
+
+        This method supports partial replacement: each arg can accept a single
+        value (which will replace all existing values) or a mapping (which will
+        replace matching entries in the old object). Otherwise, all fields are
+        the same as those used in the constructor.
+
+        Args:
+        gate_times_ns: float or Dict[Type[cirq.Gate], float].
+        t1_ns: float or Dict[cirq.Qid, float].
+        tphi_ns: float or Dict[cirq.Qid, float].
+        readout_errors: Sequence or Dict[cirq.Qid, Sequence]. Converted to
+            np.ndarray if not provided in that format.
+        gate_pauli_errors: float or Dict[cirq.OpIdentifier, float].
```

Comment:
Nit: backtips around `cirq.whatever` to doc link correctly.

Reply:
Done, here and further up the file.